src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -28,26 +32,32 @@ public class TypicalPersons {
     public static final Person ALICE = new PersonBuilder().withName("Alice Pauline")
             .withAddress("123, Jurong West Ave 6, #08-111").withEmail("alice@example.com")
             .withPhone("94351253")
-            .withTags("friends").withModules("CS1101", "CS1231S", "ES2660", "MA1521", "IS1103").build();
+            .withTags("friends").withModules("CS1101", "CS1231S", "ES2660", "MA1521", "IS1103")
+            .withGithub("alicePau").withTelegram("APauline").build();
     public static final Person BENSON = new PersonBuilder().withName("Benson Meier")
             .withAddress("311, Clementi Ave 2, #02-25")
             .withEmail("johnd@example.com").withPhone("98765432")
-            .withTags("owesMoney", "friends").build();
+            .withTags("owesMoney", "friends").withGithub("bensonnn")
+            .withTelegram("bensonnn").withGithub("benson").withTelegram("bensonmeier").build();
     public static final Person CARL = new PersonBuilder().withName("Carl Kurz").withPhone("95352563")
-            .withEmail("heinz@example.com").withAddress("wall street")
+            .withEmail("heinz@example.com").withAddress("wall street").withGithub("Carl Kurz").withTelegram("KarlCurz")
```

Comment:
I believe that github username should not have spacing 
```suggestion
            .withEmail("heinz@example.com").withAddress("wall street").withGithub("CarlKurz00").withTelegram("KarlCurz")
```